README.md

```diff
@@ -115,13 +115,31 @@ FROM plone/plone-backend:6.0.0b2
 RUN ./bin/pip install "relstorage==3.4.5" "psycopg2==2.9.3 --use-deprecated legacy-resolver"
 ```
 
-Also create a `requirements.txt` file, with packages to be installed:
+Make sure any Plone add-ons you are adding via `pip install` have their corresponding entry
+points declared in `setup.py` — otherwise they will not be loaded by Plone:
 
-```plain
-pas.plugin.authomatic
+```
+...
```

Comment:
Code blocks must have valid syntax for the language, else it does not highlight properly. This suggestion, or delete the `...`.

```suggestion
# ...
```